kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiSqlBaseLexer.g4

```diff
@@ -59,12 +59,15 @@ GTE : '>=' | '!<';
 LEFT_PAREN: '(';
 RIGHT_PAREN: ')';
 OR: 'OR';
+SEARCH_STRING_ESCAPE: '\\\'' '\\\\' '\\\'';
```

Comment:
should it be `'\'\\\''` to match `'\'` ?

Reply:
It looks like trino's SQL is `\'\\\'` , https://github.com/trinodb/trino/blob/09fd18bb47154b88365327983ae58cdab433719b/client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDatabaseMetaData.java#L1633